### PR TITLE
Add setup instructions to Contributing

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 ## Our Development Process
 Most of our work will be done in public directly on GitHub. There may be changes done through our internal source control, but it will be rare and only as needed.
 
+## Setup
+1. `git clone` the project
+2. update submodules by running `git submodule update --init` in the directory
```
Comment (nlutsenko) [inline CONTRIBUTING.md]: `git submodule update --init --recursive` to make sure that we have all the pieces and if we add any submodules later.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
...
+3. `bundle install` for the gems
```
Comment (nlutsenko) [inline CONTRIBUTING.md]: `... to install necessary RubyGems.`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 ## Our Development Process
 Most of our work will be done in public directly on GitHub. There may be changes done through our internal source control, but it will be rare and only as needed.
 
+## Setup
+1. `git clone` the project
```
Comment (nlutsenko) [inline CONTRIBUTING.md]: `Clone the project locally using 'git clone'.`

File: CONTRIBUTING.md (hunk 1 above)
Comment (nlutsenko) [inline CONTRIBUTING.md]: Also, technically - it's updating submodules, so can we replace `update` with `Fetch required submodules`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
...
+2. update submodules by running `git submodule update --init` in the directory
+3. `bundle install` for the gems
+4. `pod install` to fetch OCMock
```
Comment (nlutsenko) [inline CONTRIBUTING.md]: `Run pod install to fetch OCMock for testing.`

**Comment by nlutsenko** [comment]
Sorry, I am nitpicking here - it looks good, these are just suggestions on the language/details.
Also, maybe use a bulleted list instead of numbered? Makes adding a step in-between way easier.

**Comment by AndrewSB** [comment]
Don't apologize! I agree with all of your edits, not as experienced as you with writing good documentation

**Comment by nlutsenko** [comment]
Perfect! Thanks for making the changes.
Will merge in after commits are squashed into one.

**Comment by AndrewSB** [comment]
squashed

**Comment by nlutsenko** [comment]
Whee! Awesome, merging!

**Pull request merged**